README.md

```diff
@@ -74,7 +74,17 @@ The tool takes several options:
    - `heapoob`: Malloc OOB checker
    - `concroob`: Negative index OOB checker
    - `userinput`: User input checker
+   - `uaf` : Use After Free checker
 
+# Threading
+
+It seems Sys runs as many threads as cores you have. You can limit it here
+
+src/Static/Check.hs via setNumCapabilities N.
+
+# Getting Bitcode (LLVM)
+
+You can use wllvm or gllvm to get Bitcode and Assembly Language Format Files (*.ll)
```

Comment:
This is a nice add, I'd make it part of a separate PR. I have no idea what these are so links to them and how to use vs. say our `dclang` would be great.